.changeset/piano-hat-priority

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Unsure if it matters, but this filename is missing the `.md` file extension

Reply:
Thanks @joshfarrant